.travis.yml

```diff
@@ -16,3 +16,14 @@ script:
   - yarn lint
   - yarn test
   - yarn coveralls <./coverage/lcov.info
+
+jobs:
+  # include:
+  allow_failures:
```

Comment:
That was for testing right?

Reply:
no, it doesn't work since I don't have permission to publish. so using `allow_failures` to make CI happy for now

Reply:
Ah I understand :+1: